go.mod

```diff
@@ -77,3 +77,5 @@ require (
 	sourcegraph.com/sourcegraph/appdash v0.0.0-20180531100431-4c381bd170b4
 	sourcegraph.com/sourcegraph/appdash-data v0.0.0-20151005221446-73f23eafcf67
 )
+
+go 1.13
```

Comment:
If we actually add go 1.13 support in this PR, please add description in the PR title `Add go 1.13 support`

Reply:
It is automatically added when I use go1.13 compiler and does not take any effect on go1.12.